CenterForOpenScience/osf.io
# Feature/Remove Legacy Dashboard and Landing Pages [PLAT-877]

## Purpose

Dead code is confusing.

## Changes

Remove all of the legacy code associated with the dashboard/landing pages.

## QA Notes

Make sure logging in/logging out works as expected:
- When logged in, if manually navigating to `/goodbye/`, you are redirected to your dashboard.
- Logging out adds query_param '?goodbye=true' with alert message at top
- Logging in takes you to your dashboard.
- Manually navigating to index '/', while logged in takes you to your dash
- If on institution landing page and navigating to index, stays on institution landing page.

Make sure navigating to landing page, dashboard pages work as expected.

## Documentation
None needed.

## Side Effects



## Ticket

https://openscience.atlassian.net/browse/PLAT-877
